### PR TITLE
[TC-515] - TPv2 - adds human readable ds protocols and qstringignores to ds table

### PR DESCRIPTION
**Pull request by mitchell852**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/36/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/74/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/37/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/75/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/3/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/38/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/40/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/78/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/40/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-traffic_ops-test-PR/42/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/80/

**Comment by mitchell852** [comment]
@dneuman64 will be looking at this one

**Comment by dneuman64** [comment]
why does asfgit have to comment on this so much?  makes too much clutter

**Comment by dneuman64** [comment]
@mitchell852 I will get this merged today

File: traffic_portal/app/src/common/modules/header/header.tpl.html [hunk 1]
```diff
@@ -61,11 +61,11 @@
                     </ul>
                 </div>
             </li>
-            <li>
-                <div class="btn-group" title="Download DB">
-                    <button type="button" class="btn btn-link" ng-click="dumpDB()"><i class="fa fa-download"></i></button>
-                </div>
-            </li>
+            <!--<li>-->
```
Comment (dneuman64) [inline traffic_portal/app/src/common/modules/header/header.tpl.html]: did you mean to leave this commented out?

Comment (mitchell852) [inline traffic_portal/app/src/common/modules/header/header.tpl.html]: yea, i did

**Comment by hbeatty** [comment]
GH issue #993

**Pull request closed without merging**